pallets/collective/src/migrations/v4.rs

```diff
@@ -0,0 +1,148 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Why do we need the migration? Does collective already exist in the chain storage?

Reply:
It doesn’t, leftover from forking the pallet. I believe migrations are only triggered when specified essentially.

Reply:
> It doesn’t, leftover from forking the pallet. I believe migrations are only triggered when specified essentially.

Definitely should remove then